test/webaudio/audioworklet_test_shared.inc

```diff
@@ -83,13 +93,42 @@ void initialised(EMSCRIPTEN_WEBAUDIO_T context, bool success, void* data) {
 }
 
 // Common entry point for the mixer tests
-int main() {
-  static char workletStack[AUDIO_STACK_SIZE];
+// stackSize - audio worklet stack size in bytes
+// return true if the requested memory could be allocated
+bool _main_(int stackSize) {
+  // Optional empty space before the audio worklet's stack
+#if TEST_OFFSET_GB > 0
+  char* const emptySpace = malloc(1073741824L * TEST_OFFSET_GB);
+  if (emptySpace) {
```

Comment:
Just use `assert(emptySpace)` here? The errors produced by assert are nice because they include the exact line number.